src/schema/rules/sidecars.yml

```diff
@@ -85,19 +87,215 @@ MTParameters:
+                directions](#case-4-multiple-phase-encoded-directions-pepolar)).
+        MixingTime: RECOMMENDED
+
+MRITimingParameters:
+    selectors:
+    - modality == "MRI"
+    fields:
+        EchoTime:
+            level: RECOMMENDED
+            level_addendum: |
+                REQUIRED if corresponding fieldmap data is present,
+                or the data comes from a multi-echo sequence or Arterial Spin Labeling.
+        InversionTime: RECOMMENDED
+        SliceTiming:
+            level: RECOMMENDED
+            level_addendum: |
+                REQUIRED for sparse sequences that do not have the `DelayTime` field set,
+                and Arterial Spin Labeling with `MRAcquisitionType` set on `2D`.
+        SliceEncodingDirection: RECOMMENDED
+        DwellTime: RECOMMENDED
+
+SliceTimingASL:
+    selectors:
+    - modality == "MRI"
+    - datatype == "perf"
+    - suffix in ["asl", "m0scan"]
+    - sidecar.MRAcquisitionType == "2D"
+    fields:
+        SliceTiming: REQUIRED
+
+# This is technically for sparse sequences only, but I don't know how to encode that.
+SliceTimingSparse:
+    selectors:
+    - modality == "MRI"
+    # NOTE: This is my idea for "field is defined", but I don't know if it's the best approach.
+    - sidecar.DelayTime != "n/a"
+    fields:
+        SliceTiming: REQUIRED
+
+MRIRFandContrast:
+    selectors:
+    - modality == "MRI"
+    fields:
+        FlipAngle:
+            level: RECOMMENDED
+            level_addendum: REQUIRED if LookLocker is set to `true`.
+        NegativeContrast: OPTIONAL
+
+MRIFlipAngleLookLocker:
+    selectors:
+    - modality == "MRI"
+    - sidecar.LookLocker == "true"
+    fields:
+        FlipAngle: REQUIRED
+
+MRISliceAcceleration:
+    selectors:
+    - modality == "MRI"
+    fields:
+        MultibandAccelerationFactor: RECOMMENDED
+
+MRIAnatomicalLandmarks:
+    selectors:
+    - modality == "MRI"
+    fields:
+        AnatomicalLandmarkCoordinates__mri: RECOMMENDED
+
+MRIEchoPlanarImagingAndB0Mapping:
+    selectors:
+    - modality == "MRI"
+    fields:
+        B0FieldIdentifier: RECOMMENDED
+        B0FieldSource: RECOMMENDED
+
+MRIInstitutionInformation:
+    selectors:
+    - modality == "MRI"
+    fields:
+        InstitutionName:
+            level: RECOMMENDED
+            addendum: Corresponds to DICOM Tag 0008, 0080 `InstitutionName`.
+        InstitutionAddress:
+            level: RECOMMENDED
+            addendum: Corresponds to DICOM Tag 0008, 0081 `InstitutionAddress`.
+        InstitutionalDepartmentName:
+            level: RECOMMENDED
+            addendum: Corresponds to DICOM Tag 0008, 1040 `Institutional Department Name`.
+
+# Anatomy imaging data
+
+MRIAnatomyCommonMetadataFields:
+    selectors:
+    - modality == "MRI"
+    - datatype == "anat"
+    fields:
+        ContrastBolusIngredient: OPTIONAL
+        RepetitionTimeExcitation: OPTIONAL
+        RepetitionTimePreparation: OPTIONAL
+
+# Entities
```

Comment:
I added these because they weren't covered in #1014, but I'm not sure if they should go in this file.

Reply:
This feels like it should be its own file, but it does need to be highest priority.

Only issue I'm seeing is that these will correspond to data files, and not, say, an events.tsv.